jekyll/_cci2/deployment-integrations.md

```diff
@@ -192,9 +192,30 @@ workflows:
           container-image-name-updates: 'container=${MY_APP_PREFIX}-service,tag=${CIRCLE_SHA1}'
 ```
 
-For more detailed information about the AWS ECS & AWS ECR orbs, refer to the following Orb registry pages:
+
+##### AWS CodeDeploy
+
+This orb enables you to run deployments through AWS CodeDeploy
```

Comment:
add punctuation

Reply:
change "This" to The following or the `aws-code-deploy` orb

Reply:
Done!